docs/EN/resources/FAQ.md

```diff
@@ -0,0 +1,195 @@
+ 
+The project started with Ivan Valkou's FreeAPS X implementation of the OpenAPS algorithm for iPhone and was later forked and rebranded as iAPS. Since then, many developers have contributed substantial work, leading to new features and enhancements.
+
+Following the release of version 3.0.0, the project’s direction significantly changed due to differing views on development, open source, and peer review. This led to the separation from the iAPS repository and the birth of Trio as a distinct entity. This transition marks the project’s new phase, symbolizing its evolution and the collaborative development’s dynamic nature.
+
+:::{warning}
+_Trio is a DIY, open-source community project. As such, the user must build and maintain their app with the support of the Trio community. Trio has not undergone evaluation by any regulatory agency and is for use at your own risk._ 
+:::
+
+### Can I download Trio from the App Store?
+Trio is not available to download from the App Store. Trio is an open-source, do-it-yourself automated insulin delivery application. Community members who wish to utilize Trio must build it themselves. 
+
+While building your application may initially seem daunting, information and support are abundant and available to help you progress down this pathway. Each step required to build Trio can be found within this documentation. Additional support can be found on the Trio [Facebook](https://www.facebook.com/groups/diytrio) group and [Discord](https://discord.com/invite/FnwFEFUwXE) group. 
+
+### Do I have to be "tech-smart" to build Trio?
+No. You do not need any experience in code or computers to build Trio. If you own a computer or tablet and an iPhone, you already have the required experience. Beyond that, read the directions slowly and diligently. All the information you will need is in these documents.
+
+Often, the non-tech people do better than the tech people in building Trio. Why? Because the non-tech people take the time to read slowly and look at the screenshots in the directions. The tech people often skim and miss steps, which then leads to build errors that have to be retraced and fixed.
+
+### How long does it take to build Trio?
+The answer to this question is very user-dependent. If you are new to building and need to immerse yourself in the documentation, then a few hours. However, some users like to break this down into smaller, more manageable chunks and complete the task over 24 hours. If you are a repeat builder familiar with the documentation, build, and update process, then 10 - 15 minutes. 
+There are two options for building at present:
+* The 'Browser Build' method using GitHub.
+* The 'Mac Build' method using XCode.
+
+### Do I need to own an Apple computer?
+You no longer need to own an Apple computer. If you chose the 'Mac Build' method, you still don't have to own an Apple computer, but you must at least borrow one.  
```

Comment:
Let's add a statement here that clarifies a mac is not needed for Browser Build through GitHub.

Reply:
All sorted